Controller/SecurityFOSUser1Controller.php

```diff
@@ -80,12 +80,10 @@ public function loginAction()
             ? Security::LAST_USERNAME : SecurityContextInterface::LAST_USERNAME;
 
         // NEXT_MAJOR: Symfony <2.4 BC. To be removed.
-        if ($this->has('security.csrf.token_manager')) {
-            $csrfToken = $this->get('security.csrf.token_manager')->getToken('authenticate')->getValue();
+        if ($this->container->has('security.csrf.token_manager')) {
+            $csrfToken = $this->container->get('security.csrf.token_manager')->getToken('authenticate')->getValue();
         } else {
-            $csrfToken = $this->has('form.csrf_provider')
-                ? $this->get('form.csrf_provider')->generateCsrfToken('authenticate')
-                : null;
+            $csrfToken = $this->container->get('form.csrf_provider')->generateCsrfToken('authenticate');
```

Comment:
This is the original code changed: https://github.com/sonata-project/SonataUserBundle/pull/845/files#diff-0f7ec6fc41c7f2a56dd9389f2841d511L81